modules/red-hat-quay-builders-ui.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
+[id="creating-a-build-trigger"]
+= Creating a build trigger
+
+Use the following procedure to create a _build trigger_ using a custom Git repository.
+
+[NOTE]
+====
+The following procedure assumes that you have not included Github credentials in your `config.yaml` file.
```

Comment:
Here should be 
```
The following procedure assumes that you have included Github credentials in your `config.yaml` file.
```

Reply:
@hu-weihua Thanks. Question:

I removed this note. 

Should this procedure [here](https://github.com/quay/quay-docs/pull/1111/files#diff-4bfd23a8e3ba1b42c87f903ee83f39b4914b068ca97c2fbf765062fe8bf406efR3) show an example config.yaml file with Github credentials? Does it require GH credentials?

ty